README.md

```diff
@@ -116,6 +117,26 @@ type DatabaseService = {
 };
 ```
 
+```ts
+type MongoService = {
+  type: 'MONGO';
+  serviceName: string;
+  environment?:
+    | aws.ecs.KeyValuePair[]
+    | ((services: Services) => aws.ecs.KeyValuePair[]);
+  secrets?: aws.ecs.Secret[] | ((services: Services) => aws.ecs.Secret[]);
+  port: pulumi.Input<number>;
+  size?: pulumi.Input<Size>;
+  taskExecutionRoleInlinePolicies?: pulumi.Input<
+    pulumi.Input<RoleInlinePolicy>[]
+  >;
+  taskRoleInlinePolicies?: pulumi.Input<pulumi.Input<RoleInlinePolicy>[]>;
```

Comment:
I think that these 2 are not needed for mongo service, what do you think?

Reply:
I agree